enterprise/ha/src/test/java/slavetest/TestInstanceJoin.java

```diff
@@ -71,8 +70,8 @@ public void makeSureSlaveCanJoinEvenIfTooFarBackComparedToMaster() throws Except
             slave.shutdown();
 
             long nodeId = createNode( master, key, value );
-            createNode( master, "something", "unimportant" );
             // Rotating, moving the above transactions away so they are removed on shutdown.
+            rotateLog( master );
             master.getDependencyResolver().resolveDependency(LogRotation.class).rotateLogFile();
```

Comment:
Rather I forgot to remove the `master.getDependencyResolver().resolveDependency(LogRotation.class).rotateLogFile();` line. I removed the above `createNode` since previously rotation happened before writing to the log, that's why you needed two calls to rotate once, even on small log size threshold. Not that's no longer the case.

I'll remove the "other" log rotation line.